test-status/api.js

```diff
@@ -20,25 +20,26 @@ const {getCheckRunId, getPullRequestSnapshot} = require('./db');
  * Create a parameters object for a new status check line.
  *
  * @param {string} pullRequestSnapshot pull request snapshot from database.
- * @param {string} type tests type slug.
+ * @param {string} type major tests type slug (e.g., unit, integration).
+ * @param {string} subType sub tests type slug (e.g., saucelabs, single-pass).
  * @param {string} status status action.
  * @return {!Object} a parameters object for github.checks.create|update.
  */
-function createNewCheckParams(pullRequestSnapshot, type, status) {
+function createNewCheckParams(pullRequestSnapshot, type, subType, status) {
   const params = {
     owner: pullRequestSnapshot.owner,
     repo: pullRequestSnapshot.repo,
-    name: `ampproject/tests/${type}`,
+    name: `ampproject/tests/${type} (${subType})`,
     head_sha: pullRequestSnapshot.head_sha,
   };
   switch (status) {
     case 'queued':
       Object.assign(params, {
         status: 'queued',
         output: {
-          title: `${type} tests are queued on Travis`,
-          summary: `The ${type} tests are queued to run on Travis. Watch ` +
-            'this space for results in a few minutes!',
+          title: 'Tests are queued on Travis',
+          summary: `The ${type} (${subType}) tests are queued to run on ` +
```

Comment:
Grammar nit: Move the word `tests` next to `${type}` here, and in all other messages in this PR. IMO, it's better to read...
```
The unit tests (local-changes) are queued to run on Travis
```
... than...
```
The unit (local-changes) tests are queued to run on Travis
```

Reply:
Done